src/main/java/edu/handong/analysis/HGUCoursePatternAnalyzer.java

```diff
@@ -54,7 +54,7 @@ public void run(String[] args) {
 	 * @return
 	 */
 	private Student[] initiateStudentArrayFromLines(String[] lines) {
-		
+		//여기는 ,와공백으로 이루어 진 부분을 활용하여 구분해서 수업과 학생 정보를 받는다.	

```

Comment:
Please, avoid Korean comment as it may cause compile or build errors when the encoding setting is not proper.